### ISSUE TITLE
Building OpenNN

### ISSUE THREAD

**Issue by feco-m** [Artelnics/opennn#111]
Hello, I have some problems bulding OpenNN in Visual Studio.
After creating the files using "cmake .", I opened the .sln, I selected Release as build type and when I try to build the opennn solution I get 11 errors like this:
`Error	LNK1181	no se puede abrir el archivo de entrada '..\..\opennn\Release\opennn.lib'	leukemia	C:\Users\x\Desktop\Visual Studio\opennn-master\examples\leukemia\LINK	1	
Error	LNK1181	no se puede abrir el archivo de entrada '..\..\opennn\Release\opennn.lib'	logical_operations	C:\Users\x\Desktop\Visual Studio\opennn-master\examples\logical_operations\LINK	1`
The library opennn.lib is in the \opennn\Release\opennn.lib path.

Can you help me? Thanks in advance.

**Comment by shleym2000** [comment]
I ignored cmake in this project altogether.
Instead I opened an empty C++ library projects and added files from **opennn** sub-directory to it.
That built OnenNN library for me.

Then I had created another empty C++ application project and added example's files there. After making sure this example project will use opennn sub-directory for includes and library file from the OnenNN library project as input for linker it all works.
